en/phones.ftl

```diff
@@ -0,0 +1,157 @@
+phone-onboarding-step3-code-success-subhead-body = Choose your phone number mask and start using { -brand-name-firefox-relay } to protect your true phone number.
+phone-onboarding-step3-code-success-cta = Search for phone number masks
+phone-onboarding-step3-loading = Based on your true phone number, { -brand-name-relay } is looking for similar number combinations available to you.
+
+phone-onboarding-step4-country = United States and Canada
+phone-onboarding-step4-body = These available phone number masks are similar to your true phone number.
+phone-onboarding-step4-sub-body = Once you register a phone number mask, you cannot change it.
+phone-onboarding-step4-smiliar-phone = Similar to { $phone_number }
+phone-onboarding-step4-input-search = Enter an area code
+phone-onboarding-step4-button-more-options = Show me other options
+phone-onboarding-step4-button-register-phone-number = Register phone number mask
+phone-onboarding-step4-search-results-body = Phone number masks available in <strong>{ $location }</strong>. Once you register a phone number mask, you cannot change it.
+phone-onboarding-step4-confirm-message = To confirm your your phone number mask, enter it below. This can’t be changed later.
+phone-onboarding-step4-confirm-alt-cancel = Cancel
+phone-onboarding-step4-body-confirm-relay-number = Please confirm that this is the phone number mask you want. This can’t be changed later.
+phone-onboarding-step4-button-confirm-relay-number = Confirm
+phone-onboarding-step4-code-success-title = Congratulations!
+phone-onboarding-step4-code-success-body = You’ve registered your new phone number mask.
+phone-onboarding-step4-code-success-subhead-title = What’s next?
+phone-onboarding-step4-code-success-subhead-body-p1 = { -brand-name-relay } sent you a text with a new contact card through which we’ll forward your calls and messages.
+phone-onboarding-step4-code-success-subhead-body-p2 = Please save the contact so you can identify your forwarded messages and calls.
+phone-onboarding-step4-code-success-cta = Continue
+phone-onboarding-step4-results= No results found. Please try again.
+
+## Phone Settings
+
+phone-settings-caller-sms-log = Caller and texts log
+phone-settings-caller-sms-log-description = Allow { -brand-name-firefox-relay } to keep a log of your callers and text senders.
+phone-settings-caller-sms-log-warning = If you decide to opt out from this preference, you will lose the ability to block and reply to senders or callers. If you’ve blocked any, they will become unblocked and your existing caller and text sender log will be permanently cleared from your history.
+
+## Phone Resend SMS Banner
+
+phone-banner-resend-welcome-sms-cta = Resend welcome text
+phone-banner-resend-welcome-sms-title = Quick Tip
+phone-banner-resend-welcome-sms-body =  Remember to save the contact we shared with you by text to help you identify messages forwarded by { -brand-name-relay }. Can’t find it?
```

Comment:
Is there anything after `Can't find it?`

Reply:
Yeah, but it's on a separate line it looks like:

![image](https://user-images.githubusercontent.com/4251/193734617-80731f16-e432-4613-a142-cca6d27863c0.png)

I added a comment to clarify that in 2a33745, is that enough? I'm not sure if I have a good way of incorporating those into the same string.